src/templates/contact-us/ContactUsHeader.jsx

```diff
@@ -1,22 +1,66 @@
 import PropTypes from "prop-types"
 import { forwardRef } from "react"
 
+import editorStyles from "styles/isomer-cms/pages/Editor.module.scss"
+
 import Breadcrumb from "templates/pageComponents/Breadcrumb"
 
+import { getClassNames } from "utils"
+
 const TemplateContactUsHeader = forwardRef(({ agencyName }, ref) => (
-  <section className="bp-section" ref={ref}>
-    <div className="bp-container">
-      <div className="row is-inverted">
-        <div className="col is-8 is-offset-2">
+  <section
+    className={getClassNames(editorStyles, ["bp-section"], ["bp-section"])}
+    ref={ref}
+  >
+    <div
+      className={getClassNames(
+        editorStyles,
+        ["bp-container"],
+        ["bp-container"]
+      )}
+    >
+      <div
+        className={getClassNames(
+          editorStyles,
+          ["row", "is-inverted"],
+          ["row", "is-inverted"]
+        )}
+      >
+        <div
+          className={getClassNames(
+            editorStyles,
+            ["col", "is-8", "is-offset-2"],
+            ["col", "is-8", "is-offset-2"]
+          )}
+        >
```

Comment:
could i check why the styles are repeated twice? are there some styles globally that are not repeated within scope? (if so, can't we just paste them into local scope and delete teh global one?)

Reply:
Unfortunately, we do use the global styles elsewhere as well (editPage and associated components), and we have a relatively large number of additional such components - I've opted to leave it as is in the interest of time

Reply:
See above for the discussion on editorStyles and our solution!